buildpipeline/DotNet-CoreFx-Trusted-Linux-Crossbuild.json

```diff
@@ -499,10 +465,6 @@
     "VsoPassword": {
       "value": null,
       "isSecret": true
-    },
-    "AgentToolsPath": {
```

Comment:
This is unrelated and should be kept. @JohnTortugo just added these recently.

Reply:
Nevermind I just noticed that it just got moved and not removed.

Reply:
It just got moved.

Reply:
VSTS likes to sort them alphabetically and that seemed reasonable to me.